src/ripple/shamap/SHAMap.h

```diff
@@ -82,7 +82,7 @@ class SHAMap
     Family&                         f_;
     beast::Journal                  journal_;
     std::uint32_t                   seq_;
-    std::uint32_t                   ledgerSeq_; // sequence number of ledger this is part of
+    std::uint32_t mutable           ledgerSeq_; // sequence number of ledger this is part of
```

Comment:
I'm trying to understand what `ledgerSeq_` is really used for and if we can justify making it `mutable`. It only seems to be used in `fetchNodeFromDB` to determine if we need to fetch missing nodes. It's almost like it's a proxy object for something. I don't necessarily object to making this `mutable`, but I want to understand what's going on first.